src/modules/pos/validator/hooks/useCurrentCommissionPercentage.js

```diff
@@ -8,7 +8,7 @@ export const useCurrentCommissionPercentage = (address) => {
   const config = { params: { address: address ?? currentAccount?.metadata?.address } };
   const { data, ...others } = useValidators({ config });
   const currentCommission = useMemo(
-    () => convertCommissionToPercentage(data?.data[0].commission),
+    () => convertCommissionToPercentage(data?.data[0]?.commission),
```

Comment:
Yes, would be better to just have `commission` probably. This change i made is just to prevent a crash on page refresh, want to limit changes to only bugs at this stage.